PresentMon.props

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup Label="UserMacros">
-    <!-- PresentMonVersion should be "dev" on main branch, and the version number (e.g., 2.1.0) on release branches. -->
-    <PresentMonVersion>dev</PresentMonVersion>
+    <!-- PresentMonVersion should be "dev" on main branch, and the version number (e.g., 2.2.0) on release branches. -->
```

Comment:
Let's talk about this one. It's a philosophical one. I'm okay changing it but lets just discuss.